soh/.clang-format

```diff
@@ -1,23 +1,25 @@
-IndentWidth: 4
-Language: Cpp
-UseTab: Never
-ColumnLimit: 120
-PointerAlignment: Left
-BreakBeforeBraces: Attach
-SpaceAfterCStyleCast: false
-Cpp11BracedListStyle: false
-IndentCaseLabels: true
-BinPackArguments: true
-BinPackParameters: true
 AlignAfterOpenBracket: Align
+AlignConsecutiveBitFields: AcrossComments
+AlignConsecutiveDeclarations: AcrossComments
+AlignConsecutiveMacros: AcrossComments
```

Comment:
I found `AcrossEmptyLinesAndComments` to be quite aggressive, i think this gives us a bit more flexibility in our formatting